cmd/main.go

```diff
@@ -305,30 +305,30 @@ func main() {
 	} else {
 		crdUtils = controllers.NewDummyCRDUtils()
 	}
+	if Options.EnableDeregisterInactiveGC || Options.EnableDeletedUnregisteredGC {
```

Comment:
Is it really needed when you have the same validation below?

Reply:
We could add a comment to avoid people wondering why this check exists :)

Reply:
+1